src/Features/Core/Portable/UseAutoProperty/AbstractUseAutoPropertyAnalyzer.cs

```diff
@@ -317,34 +317,22 @@ private void Process(AnalysisResult result, SemanticModelAnalysisContext context
 
             // Place the appropriate marker on the field depending on the user option.
             var diagnostic1 = Diagnostic.Create(
-                GetFieldDescriptor(option), nodeToFade.GetLocation(),
-                additionalLocations: additionalLocations);
+                UnnecessaryWithSuggestionDescriptor,
```

Comment:
:memo: This change causes the code to always use `Unnecessary` tag. Previously the code would not gray out if the severity was `suggestion` or higher.